scripts/cc-cli/src/utils/promptContinue.ts

```diff
@@ -5,9 +5,13 @@ export async function promptContinue() {
     type: "confirm",
     name: "confirm",
     message: "Continue?",
-    initial: true,
+    initial: false,
```

Comment:
Q: what does `initial: true` mean ?

Reply:
CTRL+C return undefined you can use === to distinct undefined from false